API/projectClass/projectClass.m

```diff
@@ -777,7 +777,7 @@
             
             % parse the layers details
             layersValues = layersCell(:,2:end);
-            paramNames = string(paramStruct.paramNames);
+            paramNames = char(paramStruct.paramNames);
```

Comment:
I think `paramNames` is converted to char in `parametersClass.toStruct` ? so this might not be needed

Reply:
This changed has broken the repo. Sorry I did not mean removed the line I meant remove the char conversion